mpas_analysis/ocean/nino34_timeseries.py

```diff
@@ -0,0 +1,198 @@
+from ..shared.plot.plotting import nino34_timeseries_plot, nino34_spectra_plot
+import pandas as pd
+import numpy as np
+from scipy import signal, stats
+
+from ..shared.io import NameList, StreamsFile
+from ..shared.io.utility import buildConfigFullPath
+from ..shared.climatology import climatology
+
+from ..shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+
+from ..shared.timekeeping.utility import get_simulation_start_time, \
+    date_to_days, days_to_datetime
+
+
+def nino34_timeseries(config, streamMap=None, variableMap=None, SSTregions=None):
```

Comment:
Please wrap with `# {{{` and `return # }}}` for folds in all functions in this file.